npm-scripts.js

```diff
@@ -178,9 +189,19 @@ function replaceVersion()
 	}
 }
 
+function updateNodeDeps()
```

Comment:
This one doesn't update dependencies though, it installs those that were already present and updates lock file. Maybe have a better name for it?

Reply:
Right... on it.

Reply:
"install node deps" instead of "update node deps". Pushing.

Reply:
Done here https://github.com/versatica/mediasoup/pull/954/commits/1e457eafef4d3335c055d3d9d1c9d825f7ce309b